common/src/main/java/com/wynntils/functions/InventoryFunctions.java

```diff
@@ -19,6 +21,50 @@
 import net.minecraft.world.item.ItemStack;
 
 public class InventoryFunctions {
+    public static class CappedAccessoryDurabilityFunction extends Function<CappedValue> {
```

Comment:
```suggestion
    public static class AccessoryDurabilityFunction extends Function<CappedValue> {
```

I don't think you need to include Capped in the name. Once again, it was used in some places when we introduced capped variables to distinguish them from the old version, but for new functions, this is just making the function name more verbose.